addons/event_sale/tests/test_event_sale.py

```diff
@@ -365,6 +365,20 @@ def test_event_sale_free_full_ticket_no_confirm(self):
         with self.assertRaises(ValidationError):
             editor.action_make_registration()
 
+    def test_event_sale_stat_button(self):
+        """ Test that the sale amount displayed on the stat button matches the sale amount shown
+        when the stat button is clicked.
```

Comment:
Maybe more specific to technical
```suggestion
        """ Test that the `sale_price_total` matches the total amount of all concerned orders.
```